src/app/products/product-card/product-card.component.scss

```diff
@@ -18,25 +18,3 @@
     line-height: 1.3em;
     height: 3.9em;
   }
-
```

Comment:
You clean-up the .scss file, but I don't see any cleanup in product-card.component.ts file, make sure you removed all not used parts from there.